packages/remirror__extension-link/src/link-extension.ts

```diff
@@ -237,9 +255,7 @@ export class LinkExtension extends MarkExtension<LinkOptions> {
             }
 
             const href = node.getAttribute('href');
-            const auto =
-              node.hasAttribute(AUTO_ATTRIBUTE) ||
-              this.options.autoLinkRegex.test(node.textContent ?? '');
+            const auto = node.hasAttribute(AUTO_ATTRIBUTE);
```

Comment:
This is parsing an anchor tag already, we shouldn't indicate it is an auto link in my opinion?

Reply:
It might not a big difference here. I'll go with your choice.